addons/popochiu/engine/objects/clickable/popochiu_clickable.gd

```diff
@@ -337,17 +353,20 @@ func _on_input_event(_viewport: Node, event: InputEvent, _shape_idx: int):
 			if I.active:
 				on_item_used(I.active)
 			else:
-				handle_command(event_index)
-				
-				times_clicked += 1
+				if event.double_click:
+					handle_command(event_index, true)
```

Comment:
It might not be necessary to call `handle_command()` here. If only clicks (or taps) can be double, we can directly call `on_double_click()` here.